cogs/helpers/_taskValidator.py

```diff
@@ -13,29 +13,29 @@ def validate(fileName):
 
             tasks = data["tasks"]
             days = {
-                "0": "Sun",
-                "1": "Mon",
-                "2": "Tue",
-                "3": "Wednes",
-                "4": "Thurs",
-                "5": "Fri",
-                "6": "Satur",
-                "*": "any ",
+                "00": "Sun",
```

Comment:
Ah this left padding wasn't quite what I meant sorry, I will comment where I intended it to go below. These are just numerical keys they don't get outputted to the user at all. This should be reverted just to single digits here.